examples/editor-assignment/src/locations/ConfigScreen.spec.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ConfigScreen from './ConfigScreen';
+import { render } from '@testing-library/react';
+import { mockCma, mockSdk } from '../../test/mocks';
+
+jest.mock('@contentful/react-apps-toolkit', () => ({
+  useSDK: () => mockSdk,
+  useCMA: () => mockCma,
+}));
+
+describe('Config Screen component', () => {
+  it('Component text exists', async () => {
+    const configScreen = render(<ConfigScreen />);
+
+    // simulate the user clicking the install button
+    await mockSdk.app.onConfigure.mock.calls[0][0]();
+
+    expect(configScreen.getByText('Assign to Content Type Sidebar')).toBeInTheDocument();
+    expect(configScreen.getByText('Assign to Content Type Field (Short Text)')).toBeInTheDocument();
+
+    expect(configScreen.container).toMatchSnapshot();
```

Comment:
Forma 36 v4 often changes it's generated css classes, so this test will fail soon. Also, what are you trying to test here?

Reply:
We can remove this tbh, not actually needed